test/integration/measure_traffic_test.exs

```diff
@@ -0,0 +1,235 @@
+defmodule Realtime.Integration.MeasureTrafficTest do
+  use RealtimeWeb.ConnCase,
+    async: false,
+    parameterize: [%{serializer: Phoenix.Socket.V1.JSONSerializer}, %{serializer: RealtimeWeb.Socket.V2Serializer}]
```

Comment:
don't think we need to parameterize here as the serializer won't change any implementation